### PR TITLE
Fixing GetClrDbg ps1 and sh script to download latest clrdbg

### PR DESCRIPTION
**Pull request by rajkumar42**
Fixing GetClrDbg ps1 and sh script to pull down the latest versions of clrdbg binaries with NI images.

### CODE REVIEW

**Comment by rajkumar42** [comment]
@gregg-miskelly @chuckries @wesrupert

File: scripts/GetClrDbg.ps1 [hunk 1]
```diff
@@ -173,26 +122,16 @@ if (IsLatest $InstallPath $RuntimeID $VersionNumber) {
...
-    Remove-Item -Path $TempPath -Recurse -Force
+    if (Test-Path $InstallPath) {
+        Remove-Item -Path $InstallPath -Force -Recurse
+    }
+    
+    DownloadAndExtract $url $InstallPath
 
     WriteSuccessInfo $InstallPath $RuntimeID $VersionNumber
     Write-Host "Info: Successfully installed clrdbg at '$InstallPath'"
 }
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.ps1]: Do you know if the Docker folks fixed their issue with extra bytes at the end of the script?

File: scripts/GetClrDbg.sh [hunk 2]
```diff
@@ -369,39 +423,18 @@ else
...
-    __RuntimeID=
-    get_dotnet_runtime_id
     if [ -z $__RuntimeID ]; then
-        echo "Error: Unable to determine dotnet Runtime ID. Please make sure that dotnet is installed and the platform is supported. Look at https://www.microsoft.com/net/core for supported platforms."
-        exit 1
+        echo 'Info: Determining Runtime ID'
+        __RuntimeID=
+        get_dotnet_runtime_id
+        if [ -z $__RuntimeID ]; then
+            echo "Error: Unable to determine dotnet Runtime ID. Please make sure that dotnet is installed and the platform is supported. Look at https://www.microsoft.com/net/core for supported platforms."
```
Comment (gregg-miskelly) [inline scripts/GetClrDbg.sh]: I might suggest changing this error message to suggest explicitly indicating which runtime id to use.

File: scripts/GetClrDbg.ps1 (hunk 1 above)
Comment (rajkumar42) [inline scripts/GetClrDbg.ps1]: good point! 
verified the bug has been fixed from @weidazhao.

**Pull request merged**